src/main/java/online/book/store/dto/BookSearchParametersDto.java

```diff
@@ -0,0 +1,8 @@
+package online.book.store.dto;
+
+public record BookSearchParametersDto(
+        String title,
+        String author,
+        String isbn
```

Comment:
just so you know for the future: usually, it would make sense to **filter** books  by `price from` and `price to`, by several authors, and **search** by a partial match of a book title with some `searchString` that we would receive